src/containers/PageBuilder/Primitives/SearchCTA/FilterKeyword.module.css

```diff
@@ -0,0 +1,41 @@
+@import '../../../../styles/customMediaQueries.css';
+
+.root {
+  width: 100%;
+}
+
+.customField {
+  margin-bottom: 8px;
+  height: 40px;
+  display: grid;
+  align-items: center;
+  color: var(--colorGrey800);
+
+  input {
```

Comment:
No element styling!

Reply:
If you need to extend styling coming from FieldTextInput, then build the feature there
https://github.com/sharetribe/web-template/blob/main/src/components/FieldTextInput/FieldTextInput.js#L51-L55

```js
    classNames(css.input, {
      [inputClassName]: !!inputClassName, // add new prop inputClassName
      [css.inputSuccess]: valid,
      [css.inputError]: hasError,
      [css.textarea]: isTextarea,
    });
```

Then you can move all these input* element classes into a new class on this file:
```
.input { 
  /* something */
  
  &:focus {
    /* something */
  }
}
```